content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,183 @@
-  vendor.
+On the new :guilabel:`Vendor Bill`, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note`
+to add credit or debit notes to the bill; add a :guilabel:`Bill Reference` number; then click
+:menuselection:`Register Payment --> Create Payment` to complete the :guilabel:`Vendor Bill`.
 
-- The vendor is charging you for something you did not order.
+Create and manage vendor bills in Accounting
+============================================
 
+Vendor bills can also be created directly from the :guilabel:`Accounting` app, *without* having to
+create a purchase order first. To do this, go to :menuselection:`Accounting --> Vendors --> Bills`,
+and click :guilabel:`Create`.
 
+Then, add a :guilabel:`Vendor`, add products to the :guilabel:`Product` lines by clicking
+:guilabel:`Add a line`, and add a :guilabel:`Bill Date`. :guilabel:`Confirm` the bill. From here,
+click the :guilabel:`Journal Items` tab to view (or change) the :guilabel:`Account` journals that
+were populated based on the configuration on the :guilabel:`Vendor` and :guilabel:`Product` forms.
 
-How product quantities are managed
-----------------------------------
+Then, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note` to add credit or debit notes
+to the bill; add a :guilabel:`Bill Reference` number; then click :menuselection:`Register Payment
+--> Create Payment` to complete the :guilabel:`Vendor Bill`.
 
-By default, services are managed based on ordered quantities, while
-stockables and consumables are managed based on received quantities.
+.. tip::
+   To tie the :guilabel:`Draft Bill` to an existing purchase order, click the :guilabel:`drop-down`
+   menu next to :guilabel:`Auto-Complete`, and select a :abbr:`PO (Purchase Order)` from the menu.
+   The bill will auto-populate with the information from the :abbr:`PO (Purchase Order)`.
 
-If you need to manage products based on ordered quantities over received
-quantities, you will need to belong to the group **Purchase Manager**.
-Ask your system administrator to enable these access on :menuselection:`Settings
---> Users --> Users --> Access Rights`. Once you belong to the correct group,
-select the product(s) you wish to modify, and you should see a new field appear,
-labeled **Control Purchase Bills**.
+.. image:: manage/manage-auto-complete.png
+   :align: center
+   :alt: Auto-complete drop-down list on draft vendor bill.
 
-.. image:: manage/manage08.png
-  :align: center
+Batch billing
+=============
 
-You can then change the default management method for the selected
-product to be based on either:
+Vendor bills can be processed and managed in batches in the :guilabel:`Accounting` app. To do this,
+go to :menuselection:`Accounting --> Vendors --> Bills`. Then, click the :guilabel:`checkbox` at
+the top left of the page under :guilabel:`Create`. This will select all existing vendor bills with
+a :guilabel:`Posted` or :guilabel:`Draft` :guilabel:`Status`.
```

Comment:
Vendor bills can be processed and managed in batches in the :guilabel:`Accounting` app. 

To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the :guilabel:`checkbox` at
the top left of the page, beside the :guilabel:`Number` column, under the :guilabel:`Create` button. This selects all existing vendor bills with a :guilabel:`Posted` (or :guilabel:`Draft`) :guilabel:`Status`.